uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/interaction/PromptHandlerParameters.java

```diff
@@ -0,0 +1,74 @@
+//  Copyright (c) Microsoft Corporation.
+//  All rights reserved.
+//
+//  This code is licensed under the MIT License.
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files(the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and / or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions :
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+package com.microsoft.identity.client.ui.automation.interaction;
+
+import com.microsoft.identity.client.ui.automation.broker.ITestBroker;
+
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NonNull;
+
+@Builder
+@Getter
+/**
+ * A set of values that can be used to denote the behaviour we expect to observe during an oauth
+ * authorization request. These values can then be supplied to a {@link AbstractPromptHandler} and
+ * used to handle prompt in the manner as expected during the execution of a given test.
+ * The test should fail if the actual behaviour observed deviates from what is denoted as expected
+ * via these parameters.
+ */
+public class PromptHandlerParameters {
+
+    @NonNull
+    // the prompt behaviour we expect
+    private PromptParameter prompt;
+
+    // whether session is expected or not (via presence of a cookie)
+    private boolean sessionExpected;
+
+    // whether login hint was supplied or not to the interactive request
+    private boolean loginHintProvided;
+
+    // whether we expect to receive consent page or not
+    private boolean consentPageExpected;
+
+    // whether we expect to receive speed bump or not
+    private boolean speedBumpExpected;
+
+    // whether we are expecting at least one account in broker
+    private boolean expectingNonZeroAccountsInBroker;
+
+    // whether we are expecting at least one account in the browser/webview cookie
+    private boolean expectingNonZeroAccountsInCookie;
+
+    @Builder.Default
+    // The way in which we want to respond to consent page for this request
+    private UiResponse consentPageResponse = UiResponse.ACCEPT;
+
+    @Builder.Default
+    // The way in which we want to respond to speed bump page for this request
```

Comment:
Same here, comments between the annotation and its target... should be flipped around

Reply:
Addressed